### PR TITLE
Npm modules support

### LINKED ISSUE #5: Support using individual npm modules
Makes the generated file change from importing `_.last = require('lodash/array/last')` to `_.last = require('lodash.last')`

Issue would be touching the `package.json` -- perhaps inform user to run `npm i --save lodash.last`?

Could be useful for to avoid issues like https://github.com/lodash/lodash/issues/1149

### PR DESCRIPTION
**Pull request by megawac**
Fixes #5

### CODE REVIEW

**Comment by megawac** [comment]
This will error on the chain build if `create`, `mixin`, or internals aren't included

File: bin/lodash-modularize [hunk 1]
```diff
@@ -91,6 +97,8 @@ var argv = require('yargs')
 // I want it all
 require('bluebird').longStackTraces();
 var modularize = require('..');
+var inquirer = require('inquirer');
+var npm = require('npm').load();
```
Comment (d10) [inline bin/lodash-modularize]: Why not only load npm when needed?

**Comment by megawac** [comment]
I don't see around e6dc61d892fcca21f1eafc9214b1c35257fd8c92 @jdalton do you?

**Comment by jdalton** [comment]
Sorry, I'm not following the issue, what's up?

**Comment by megawac** [comment]
As far as I can tell you can't construct chaining from the published set of npm modules (e.g. there isnt' a `lodash.chain` in the npm repository)

**Pull request merged**